lib/rest/chat/v2/service/channel.d.ts

```diff
@@ -31,6 +31,24 @@ type ChannelWebhookEnabledType = 'true'|'false';
  */
 declare function ChannelList(version: V2, serviceSid: string): ChannelListInstance;
 
+/**
+ * Options to pass to remove
+ *
+ * @property xTwilioWebhookEnabled - The x_twilio_webhook_enabled
+ */
+interface ChannelInstanceDeleteOptions {
+  xTwilioWebhookEnabled?: ChannelWebhookEnabledType;
```

Comment:
I don't see the type `ChannelWebhookEnabledType` being exported.

Reply:
It's an enum defined at the top of the file. Are those usually exported?

Reply:
If it's used in an interface, then it needs to be exported so clients can use it. Could be a prior issue, just surfacing now.

Reply:
Update: turns out no Enums are ever being exported. Will introduce fix in this PR.